.github/workflows/docker.yml

```diff
@@ -2,6 +2,7 @@ on:
   push:
     branches:
       - main
+      - v07
```

Comment:
Do you think it makes sense to also include v07 to the docker image name (only for v07 branch)? To differentiate between the main docker images.

Reply:
Yes.